tests/conftest.py

```diff
@@ -58,7 +57,14 @@ def gen_random_ptraj(request):
     request.cls.times = np.array([[i, 0] for i in range(N_FRAMES)[::2]], dtype=np.int32)
 
     request.cls.ptraj = md.Trajectory(xyz=xyz, topology=None)
-    request.cls.ptraj.unitcell_vectors = np.ascontiguousarray(
-        rng.standard_normal((N_FRAMES, 3, 3)) + 2 * np.eye(3, 3),
-        dtype=np.float32,
-    )
+
+    while True:
```

Comment:
Is there a way to constrain values from RNG to give us valid unit cells (to avoid while loop)?

(not a review, just a question)

Reply:
I'm leaning to say no. Currently, the RNG just draws from the normal distribution to build the unit cell vectors (and then we transform them back to the lengths/angles to check if they are valid), so we'd have to map the domain of the angles to the unit cell vectors which seems like a lot to figure out, especially because our domain is a little complicated (4 constraints).

We could somehow build it the other way round, but then that seems like a lot of rewrite for very little gain.

From running thousands of these test, the max i would get is about 2/200 failures, so the while loop will just run 2-3 times max, if it ever gets an error the first time.